# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dvb-apps/dvb-install.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN dvb-apps/util/av7110_loadkeys/Makefile dvb-apps.work/util/av7110_loadkeys/Makefile
--- dvb-apps/util/av7110_loadkeys/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/av7110_loadkeys/Makefile	2005-03-01 13:05:34.000000000 +0100
@@ -2,8 +2,8 @@
 CFLAGS = -g -Wall -O2 $(ARCH)
 CPPFLAGS = $(DEFS)
 DEFS = '-DDATADIR="$(datadir)"'
-bindir  = /usr/local/bin
-datadir = /usr/local/share
+bindir  = /usr/bin
+datadir = /usr/share
 
 all: av7110_loadkeys evtest
 
@@ -53,6 +53,7 @@
 install: install-bin install-data
 install-bin: av7110_loadkeys
 	install -m 755 av7110_loadkeys $(DESTDIR)$(bindir)
+	install -m 755 evtest $(DESTDIR)$(bindir)
 install-data:
 	mkdir -p $(DESTDIR)$(datadir)/dvb/av7110_loadkeys
 	cp *.rc5 *.rcmm $(DESTDIR)$(datadir)/dvb/av7110_loadkeys/
diff -ruN dvb-apps/util/dvbdate/Makefile dvb-apps.work/util/dvbdate/Makefile
--- dvb-apps/util/dvbdate/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/dvbdate/Makefile	2005-03-01 13:05:06.000000000 +0100
@@ -6,7 +6,7 @@
 
 OBJS    = dvbdate.o
 TARGET  = dvbdate
-bindir  = /usr/local/bin
+bindir  = /usr/bin
 
 all: $(TARGET)
 
diff -ruN dvb-apps/util/dvbnet/Makefile dvb-apps.work/util/dvbnet/Makefile
--- dvb-apps/util/dvbnet/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/dvbnet/Makefile	2005-03-01 13:06:09.000000000 +0100
@@ -3,8 +3,8 @@
 CFLAGS  = -g -O2 -Wall $(ARCH)
 CPPFLAGS= -MD -I. -I../../include $(DEFS)
 LFLAGS  =
-bindir = /usr/local/bin
-libdir = /usr/local/lib
+bindir = /usr/bin
+libdir = /usr/lib
 
 OBJS    = dvbnet.o
 TARGET  = dvbnet
diff -ruN dvb-apps/util/dvbtraffic/Makefile dvb-apps.work/util/dvbtraffic/Makefile
--- dvb-apps/util/dvbtraffic/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/dvbtraffic/Makefile	2005-03-01 13:06:44.000000000 +0100
@@ -1,6 +1,6 @@
 CFLAGS   = -g -O2 -Wall $(ARCH)
 CPPFLAGS = -MD -I../../include $(DEFS)
-bindir = /usr/local/bin
+bindir = /usr/bin
 
 dvbtraffic: dvbtraffic.c
 	$(CC) $(CPPFLAGS) $(CFLAGS) $< -o $@
diff -ruN dvb-apps/util/scan/Makefile dvb-apps.work/util/scan/Makefile
--- dvb-apps/util/scan/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/scan/Makefile	2005-03-01 13:11:41.000000000 +0100
@@ -4,8 +4,8 @@
 CPPFLAGS = -MD -I../../include -I../lib $(DEFS)
 LFLAGS = -g -Wall
 DEFS = '-DDATADIR="$(datadir)"'
-bindir = /usr/local/bin
-datadir = /usr/local/share
+bindir = /usr/bin
+datadir = /usr/share
 
 OBJS = diseqc.o dump-zap.o dump-vdr.o scan.o ../lib/lnb.o
 SRCS = $(OBJS:.o=.c)
diff -ruN dvb-apps/util/szap/Makefile dvb-apps.work/util/szap/Makefile
--- dvb-apps/util/szap/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/szap/Makefile	2005-03-01 13:12:13.000000000 +0100
@@ -4,8 +4,8 @@
 LFLAGS = -Wall -g -O2
 RM = rm -f
 DEFS = '-DDATADIR="$(datadir)"'
-bindir = /usr/local/bin
-datadir = /usr/local/share
+bindir = /usr/bin
+datadir = /usr/share
 
 TARGETS = azap szap tzap czap femon
 OBJS = azap.o szap.o tzap.o czap.o femon.o
@@ -42,6 +42,8 @@
 	mkdir -p $(DESTDIR)$(bindir)/
 	for i in $(TARGETS) ; do cp $$i $(DESTDIR)$(bindir)/ ; done
 install-data:
+	mkdir -p $(DESTDIR)$(datadir)/dvb/
+	cp -a channels-conf/* $(DESTDIR)$(datadir)/dvb
 
 -include $(wildcard *.d) dummy
 
diff -ruN dvb-apps/util/ttusb_dec_reset/Makefile dvb-apps.work/util/ttusb_dec_reset/Makefile
--- dvb-apps/util/ttusb_dec_reset/Makefile	2000-01-01 00:00:00.000000000 +0100
+++ dvb-apps.work/util/ttusb_dec_reset/Makefile	2005-03-01 13:12:40.000000000 +0100
@@ -4,7 +4,7 @@
 CPPFLAGS = $(DEFS)
 LFLAGS	= -g -Wall
 LDFLAGS = -lusb
-bindir  = /usr/local/bin
+bindir  = /usr/bin
 
 OBJS	= ttusb_dec_reset.o
 TARGET	= ttusb_dec_reset
